{
  "schema_version": "1.2.0",
  "id": "GHSA-625g-gx8c-xcmg",
  "modified": "2022-05-14T02:09:22Z",
  "published": "2022-05-14T02:09:22Z",
  "aliases": [
    "CVE-2014-0482"
  ],
  "details": "The contrib.auth.middleware.RemoteUserMiddleware middleware in Django before 1.4.14, 1.5.x before 1.5.9, 1.6.x before 1.6.6, and 1.7 before release candidate 3, when using the contrib.auth.backends.RemoteUserBackend backend, allows remote authenticated users to hijack web sessions via vectors related to the REMOTE_USER header.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2014-0482"
    },
    {
      "type": "WEB",
      "url": "https://www.djangoproject.com/weblog/2014/aug/20/security/"
    },
    {
      "type": "WEB",
      "url": "http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html"
    },
    {
      "type": "WEB",
      "url": "http://secunia.com/advisories/59782"
    },
    {
      "type": "WEB",
      "url": "http://secunia.com/advisories/61276"
    },
    {
      "type": "WEB",
      "url": "http://secunia.com/advisories/61281"
    },
    {
      "type": "WEB",
      "url": "http://www.debian.org/security/2014/dsa-3010"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-287"
    ],
    "severity": "MODERATE",
    "github_reviewed": false
  }
}